<launch>
<!--<include file="$(find realsense2_camera)/launch/rs_t265.launch"/>-->
 <!-- nodelet manager from image stream -->
  <node pkg="nodelet" type="nodelet" name="camera_nodelet_manager"  args="manager" />
  <node pkg="nodelet" type="nodelet" name="image_proc_fisheye" args="load image_proc_fisheye/RectifyNodelet camera_nodelet_manager" output="screen">    
    <remap from="image_raw" to="/camera/fisheye1/image_raw"/>
    <remap from="camera_info" to="/camera/fisheye1/camera_info"/>
  </node>
</launch>